content/blog/2024/10/policy-groups.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Policy Groups deep dive
+authors:
+  - Víctor Cuadrado Juan
+date: 2024-11-04
+---
+
+With [v1.17](https://www.kubewarden.io/blog/2024/10/kubewarden-1-17-release/),
+we introduced a new powerful feature, Policy Groups, brought by 2 new Kubernetes
+Custom Resources:
+
+- **AdmissionPolicyGroups**: Namespaced policy comprised of several policies.
+- **ClusterAdmissionPolicyGroups**: Clusterwide policy comprised of several policies.
+
+These new Policy Groups resources define a policy comprised of several policies and
+their policy settings, and they perform a combined evaluation of those multiple
+policies using logical operators.
+
+Why are these useful? Because they reuse existing policies, reducing the need
+for custom policy creation. And they provide complex logic while at the same
+time reducing complexity as you have all the logic packed in 1 resource
```

Comment:
I suggest 'contained' as 'packed in' does not feel like reducing complexity. As in lots of things packed in. Also we try to use written numbers up to ten. Not particularly important, but a stylistic convention.
```suggestion
time reducing complexity as you have all the logic contained in one resource
```